### ISSUE TITLE
bug: Run timer stays at 00:00:00 if you reconnect in the middle of a delay

### ISSUE THREAD

**Issue by SyntaxColoring** [Opentrons/opentrons#7740]
# Overview

If the Opentrons App disconnects from an OT-2 in the middle of a protocol, and reconnects while the OT-2 is in the middle of a delay step, the Opentrons App's run timer will be stuck at `00:00:00`, as if the protocol had never started.

This may contribute to users thinking a protocol has "canceled itself."

# Steps to reproduce

1. Run this protocol on your OT-2.
2. During the 5-minute delay, close and reopen the Opentrons App.

```python
metadata = {
    'apiLevel': '2.9'
}
    
def run(protocol):
    tip_rack = protocol.load_labware('opentrons_96_tiprack_300ul', 8)
    pipette = protocol.load_instrument('p300_multi_gen2', 'left')
    
    pipette.pick_up_tip(tip_rack.wells()[0])
    pipette.drop_tip(tip_rack.wells()[0])
    protocol.delay(minutes=5)
    pipette.pick_up_tip(tip_rack.wells()[0])
    pipette.drop_tip(tip_rack.wells()[0])
```

# Current behavior

1. Before closing the Opentrons App, the timer is counting through the delay.
2. After reopening the Opentrons App, the timer will be at `00:00:00`.
3. The timer will catch up and resume counting if any of these happen:
    * You wait until the end of the delay.
    * You click **Pause** and then **Resume**.

# Expected behavior

After reconnecting, the timer should be wherever it would be if you had never disconnected in the first place.

**Comment by SyntaxColoring** [comment]
Note to self: Is this because the timer doesn't increment during pauses, and delays are implemented as pauses?

**Comment by celsasser** [comment]
I will take a look. But if anybody else more qualifier (which is everybody) wants to look into it feel free to jump in or hijack the ticket.

**Comment by SyntaxColoring** [comment]
Fixed by #7937.
